addons/mail/static/src/rtc/call_participant_card.js

```diff
@@ -22,7 +21,7 @@ export class CallParticipantCard extends Component {
         this.popover = usePopover(CallContextMenu);
         this.rpc = useService("rpc");
         this.rtc = useRtc();
-        this.store = useStore();
+        this.discussStore = useService("discuss.store");
```

Comment:
Same comment: accessing `discuss` from outside its folder is not correct.

Reply:
same comment: I'll move the whole rtc folder to discuss.